src/components/RangeSlider.react.js

```diff
@@ -152,6 +157,49 @@ RangeSlider.propTypes = {
      */
     pushable: PropTypes.oneOfType([PropTypes.bool, PropTypes.number]),
 
+    tooltip: PropTypes.oneOfType([
```

Comment:
This is a pattern that @Marc-Andre-Rivet suggested. Example usage [here](https://github.com/plotly/dash-table/blob/master/src/dash-table/dash/DataTable.js#L453) in `DataTable`. However, it's not working for me, allowing e.g.

``` python
    dcc.RangeSlider(
        id='my-range-slider',
        min=0,
        max=20,
        step=0.5,
        value=[5,10],
        tooltip={
            'foo': 'bar' # I expect this not to be allowed with the strict type-checking below
        }
    ),
```

Reply:
Huh, not sure what's going on there, looks to me like that should work.

Note though that your last one, `PropTypes.exact({visible, position})`, actually encompasses both of the other two as well as `{}` since neither is required. Unless you specifically want to prevent `tooltip={}` from creating a tooltip (and to my mind that seems like a fine way for users to say "give me a default tooltip") I'd just use that last form by itself, no `oneOfType`.

Reply:
> Unless you specifically want to prevent tooltip={} from creating a tooltip

IIRC @Marc-Andre-Rivet  _did_ want to prevent this, but I could have misunderstood him. Regardless, I'm inclined to agree `tooltip={}` should be allowed. `propTypes.exact` seems to actually allow other types as well (not just objects), which is confusing to me, but that means any truthy value will result in a default tooltip.

As an aside, I'd like to be more informed about stricter type-checking with `propTypes` and the `oneOfType` / `exact` pattern — do you understand why the above doesn't work @Marc-Andre-Rivet ?).